pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -25,7 +25,6 @@ import (
 	v2_cluster "github.com/envoyproxy/go-control-plane/envoy/api/v2/cluster"
 	"github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	"github.com/gogo/protobuf/types"
-
```

Comment:
won't removing this line cause linter issues?  or did we recently stop having separation between outside libs and istio libs in the imports?

Reply:
oh go fmt or goimports did this, on goland IDE..